tests/search/manager_test.py

```diff
@@ -0,0 +1,142 @@
+from tests.unit_test_base import UnitTestBase
+from tests.search.test_wrapper import TestWrapper as SearchTestWrapper
+from pyshelf.search.type import Type as SearchType
+from pyshelf.search.sort_type import SortType
+import tests.metadata_utils as utils
+import time
+
+
+class ManagerTest(UnitTestBase):
+    def setUp(self):
+        self.test_wrapper = SearchTestWrapper()
+        self.search_manager = self.test_wrapper.search_container.search_manager
+        self.test_wrapper.setup_metadata()
+        self.test_wrapper.setup_metadata("other", "/this/that/other", "1.1")
+        self.test_wrapper.setup_metadata("thing", "/thing", "1.2")
+        self.test_wrapper.setup_metadata("blah", "/blah", "1.19")
+        self.test_wrapper.setup_metadata("a", "/a", "1.19")
+        self.test_wrapper.setup_metadata("zzzz", "/zzzz", "1.19")
+        # temp fix
+        time.sleep(1)
+
+    def tearDown(self):
+        self.test_wrapper.teardown_metadata("test")
+        self.test_wrapper.teardown_metadata("other")
+        self.test_wrapper.teardown_metadata("thing")
+
+    def test_equality_search(self):
+        results = self.search_manager.search({
+            "search": [
+                {
+                    "field": "artifactName",
+                    "search_type": SearchType.MATCH,
+                    "value": "test"
+                },
+                {
+                    "field": "artifactPath",
+                    "search_type": SearchType.WILDCARD,
+                    "value": "tes?"
+                }
+            ]
+        })
+        expected = [utils.get_meta()]
+        self.assertEqual(results, expected)
+
+    def test_tilde_search_and_sort(self):
+        results = self.search_manager.search({
+            "search": [
+                {
+                    "field": "version",
+                    "search_type": SearchType.TILDE,
+                    "value": "1.1"
+                }
+            ],
+            "sort": [
+                {
+                    "field": "version",
+                    "sort_type": SortType.VERSION,
+                    "flag_list": [
+                        SortType.ASC
+                    ]
+                },
+            ]
+        })
+        expected = [
+            utils.get_meta("other", "/this/that/other", "1.1"),
+            utils.get_meta("thing", "/thing", "1.2")
+        ]
+        self.assertEqual(results[0:2], expected)
+        for item in results[2::]:
+            self.assertEqual(item["version"]["value"], "1.19")
```

Comment:
So because of the way these results are sorted (just by version) when there is a tie the order is variable between the ties. So I want to make sure the first two elements are correct (in order and contents) and the next 3 are the right version. Let me know if there is a better way to handle this. assertItemsEqual is only in version 2.7 and forward.